codegen/templates/manifests/crd.yamltmpl

```diff
@@ -0,0 +1,30 @@
+[[- range $crd := .Crds -]]
+[[- if and (and (lt (len $crd.Spec.Versions) 2) (string_contains (index $crd.Spec.Versions 0).Name "alpha")) (should_not_skip $crd.Spec.Group (index $crd.Spec.Versions 0).Name $.ShouldSkip) ]]
+{{- if has "[[ $crd.GetName ]]/[[ (index $crd.Spec.Versions 0).Name ]]" $.Values.enabledExperimentalApi }}
+[[- end ]]
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations: [[- $crd.GetAnnotations | toYaml | nindent 4 ]]
+  labels: [[- $crd.GetLabels | toYaml | nindent 4 ]]
+  name: [[ $crd.GetName ]]
+spec:
+  group: [[ $crd.Spec.Group ]]
+  names: [[- $crd.Spec.Names | toYaml | nindent 4 ]]
+  scope: [[ $crd.Spec.Scope ]]
+  versions:
+  [[- range $version := $crd.Spec.Versions ]]
+  [[- if and (and (gt (len $crd.Spec.Versions) 1) (string_contains $version.Name "alpha")) (should_not_skip $crd.Spec.Group $version.Name $.ShouldSkip) ]]
+  {{- if has "[[ $crd.GetName ]]/[[ $version.Name ]]" $.Values.enabledExperimentalApi }}
+  [[- end ]]
+  -
+  [[- $version | toYaml | indent 4 | trimPrefix "   " ]]
+  [[- if and (and (gt (len $crd.Spec.Versions) 1) (string_contains $version.Name "alpha")) (should_not_skip $crd.Spec.Group $version.Name $.ShouldSkip) ]]
+  {{- end }}
+  [[- end ]]
+  [[- end ]]
+[[- if and (and (lt (len $crd.Spec.Versions) 2) (string_contains (index $crd.Spec.Versions 0).Name "alpha")) (should_not_skip $crd.Spec.Group (index $crd.Spec.Versions 0).Name $.ShouldSkip) ]]
```

Comment:
Are these just to make sure new-lines are created when needed?

Reply:
Not sure what exactly you are referring to. If it's the conditional on the line above, it's to conditionally render `{{- end }}` that corresponds to the conditional here: https://github.com/solo-io/skv2/pull/462/files/acf693e11fd37032d1e9813a85d5e6888a1c0a4d#diff-033b9d4512e2683aff9f19928ec94eaaa06e46cbfcede987461da2079459436dR3